zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -102,18 +107,22 @@ use crate::{types::MetaAddr, AddressBook, BoxError, PeerAddrState, Request, Resp
 pub(super) struct CandidateSet<S> {
```

Comment:
Do you think we should also add a summary `Security` section to the `CandidateSet` doc comment?

(When we're done with all the security changes, there will be 5+ items in this section.)

Reply:
seems like a good idea for a follow up change

Reply:
That's fair enough, as long as we don't forget!